packages/nodejs/test/clientV2.test.ts

```diff
@@ -232,44 +297,50 @@ test('Failed invoke contract', async () => {
     expect(result.reason.tag).toBe(v1.RejectReasonTag.RejectedReceive);
 });
 
-test('Invoke contract on v0 contract', async () => {
-    const result = await clientV2.invokeContract(
-        {
+test.each([clientV2, clientWeb])(
+    'Invoke contract on v0 contract',
+    async (client) => {
+        const result = await client.invokeContract(
+            {
+                invoker: testAccount,
+                contract: {
+                    index: 6n,
+                    subindex: 0n,
+                },
+                method: 'PiggyBank.insert',
+                amount: new v1.CcdAmount(1n),
+                parameter: undefined,
+                energy: 30000n,
+            },
+            testBlockHash
+        );
+
+        expect(result).toEqual(expected.invokeInstanceResponseV0);
+    }
+);
+
+test.each([clientV2, clientWeb])(
+    'Invoke contract same in v1 and v2 on v1 contract',
+    async (client) => {
+        const context = {
             invoker: testAccount,
             contract: {
-                index: 6n,
+                index: 81n,
```

Comment:
I'm sure it's fine, but why change the contract?

Reply:
It is another test, the git diff is just weird. (Above I still have the test with the index 6 contract)
This one invokes a v1 contract, which the one on index 81 is, and the 6 one is a v0 contract.

Reply:
Makes sense